File: src/main/java/com/jcloisterzone/figure/neutral/Count.java
package com.jcloisterzone.figure.neutral;

import com.jcloisterzone.Immutable;
import com.jcloisterzone.board.pointer.FeaturePointer;
import com.jcloisterzone.game.capability.trait.MeteoriteProtected;

@Immutable
public class Count extends NeutralFigure<FeaturePointer> implements MeteoriteProtected {

    private static final long serialVersionUID = 1L;

    public Count(String id) {
        super(id);
    }
}

File: src/main/ts/com/jcloisterzone/figure/neutral/Count.ts
import type { FeaturePointer } from "../../board/pointer/FeaturePointer.js";
import type { MeteoriteProtected } from "../../game/capability/trait/MeteoriteProtected.js";
import { NeutralFigure } from "./NeutralFigure.js";

/** The Count of Carcassonne. */
export class Count extends NeutralFigure<FeaturePointer> implements MeteoriteProtected {
  static readonly simpleName = "Count";
  readonly meteoriteProtected = true as const;
  constructor(id: string) {
    super(id);
  }
}
